Restructure monitor command docs and add monitor partial (#28753)
hashicorp/vault: website/content/docs/commands/monitor.mdx

---
layout: docs
page_title: 'monitor - Vault CLI'
description: |-
  Display the server logs of a Vault server in real time.
---

# `monitor`

> Show a real time display of the server logs of a Vault server

Reviewer: ```suggestion
Stream Vault server logs in real-time to `stdout`.
```

Style correction: write in active voice, prefer concise language

<CodeBlockConfig hideClipboard>

```shell-session
$ vault monitor [flags]

$ vault monitor [-help | -h]
```

</CodeBlockConfig>

## Description

`vault monitor` command shows a real time display of the server logs of a Vault
server. This command accepts a log level as an argument, which can be different
from the log level that the Vault server was started with.

`vault monitor` honors the `VAULT_ADDR` environment variable. The address
specified determines the target server that will be monitored.

### Limitations and warnings

- Note that this command is designed to run indefinitely. It is similar to
  `tail -f` in the Unix world. This command will not exit on its own unless
  it encounters an unexpected error. As a user, you must terminate this
  process yourself to shut it down.

- If Vault is emitting log messages faster than a receiver can process them, the
  some log lines will be dropped.

> ## Command arguments

Reviewer: ```suggestion
<Tip title="Related API endpoints">

  MonitorLogs - [`GET:	/sys/monitor`](/vault/api-docs/system/monitor)

</Tip>

## Command arguments
```

- None

## Command options

- None

## Command flags

<br />

@include 'cli/monitor/flags/log-level.mdx'

<br />
<hr />
<br />

@include 'cli/monitor/flags/log-format.mdx'

## Standard flags

> @include 'cli/standard-settings/all-standard-flags-but-format.mdx'

Reviewer: This command might actually support the `-format` flag since it's printing info to the screen. I would change this to `all-standard-flags.mdx` for now
